openfga/v1/openfga.proto

```diff
@@ -32,6 +32,57 @@ message Object {
   ];
 }
 
+// User represents any possible value for a user (subject)
+//
+// Can either be a:
+// - Specific user object (ex: 'user:will')
+// - Specific userset (ex: 'group:engineering#member')
+// - Public-typed wildcard (ex: 'user:*')
+//
+// See https://docs.fga.dev/intro/okta-fga-concepts#what-is-a-user
+message User {
+  oneof user {
+    Object object = 1;
+    UsersetUser userset = 2;
+    TypedWildcard wildcard = 3;
+  }
+}
```

Comment:
```suggestion
// User.
//
// Represents any possible value for a user (subject). Can be a:
// - Specific user object (ex: 'user:will')
// - Specific userset (ex: 'group:engineering#member')
// - Public-typed wildcard (ex: 'user:*')
//
// See https://docs.fga.dev/intro/okta-fga-concepts#what-is-a-user
message User {
```

Which ultimately looks like this 

<img width="925" alt="image" src="https://github.com/openfga/api/assets/5374887/d3590fa2-193a-4dfc-a331-aec4ee064dbc">